### PR TITLE
[Task] Feature Detail Page Dark Theme

### PR DESCRIPTION
**Pull request by Teraaay**
## Related Links

- Task Link: https://app.asana.com/0/1202252431396579/1202381121175294/f
- Figma Link: https://www.figma.com/file/z1jx3G9usokpnHaXucuML7/Feature-Details-Page-Dark-Mode?node-id=0%3A1
- Route Link
   - Profile Feature Page: http://localhost:3000/feature/profile 
   - Match Feature Page: http://localhost:3000/feature/match
   - Chat Feature Page: http://localhost:3000/feature/chat
   - Adopt Feature Page: http://localhost:3000/feature/adopt

## Description

- Implement Dark Theme for Feature Details Pages (Refer to Figma link above for the wireframe)

## Notes

N/A

## Screenshots
Feature Details Page Dark Theme

### CODE REVIEW

**Comment by ume45876139** [comment]
# Comment
<details>
<summary>http://localhost:3000/feature/adopt</summary>

Screen Shot 2022-06-07 at 19 09 45

</details>

- [ ] Text in testimonial part is too dark

# Testing data
test device: Chrome
test time: 0.25h

**Review by ErickDoroy — APPROVED** [review]

**Review by ume45876139 — APPROVED** [review]
# Working

<details>
<summary>Browser</summary>

https://user-images.githubusercontent.com/39412730/172360458-8a7cb009-fd95-4a80-8ede-09a997e79f5e.mp4


https://user-images.githubusercontent.com/39412730/172360493-02b2298c-3099-4596-bd4a-83b4ccbd08e1.mp4


https://user-images.githubusercontent.com/39412730/172360501-ee399df8-0830-41c3-a6fb-4d51c2cc3ba9.mp4


https://user-images.githubusercontent.com/39412730/172360503-1234098d-15cb-46cc-84ee-d3359d6aa418.mp4


</details>

<details>
<summary>iPad</summary>


https://user-images.githubusercontent.com/39412730/172360696-e32fe1b7-3ce6-4360-8dc5-c67cce0dc580.mp4


</details>

<details>
<summary>iPhone</summary>

https://user-images.githubusercontent.com/39412730/172360816-77f795f3-a47d-431e-bae6-f7328cc4b03e.MP4

</details>

- Text color is correct
- There are some deformation I will separate issue

# Testing data
test device: Chrome, Safari (Mac), Firefox, Opera
test time: 0.5h
comment: 0
iteration: 0
total test time: 0.5h

File: src/pages/components/subscriptionSection/PlanCard.js [hunk 1]
```diff
@@ -3,16 +3,17 @@ import CheckCirledIcon from '../icons/CheckCircledIcon';
...
       {subscription.name}
     </div>
     <div className='rounded-b-lg'>
-      <ul className='list-none divide-y divide-orange-900 px-9 py-2'>
+      <ul className='list-none divide-y divide-orange-900 px-9 py-2 dark:divide-gray-500'>
         {subscription.content.map((feature, index) => (
           <li className='flex py-4' key={index}>
-            {feature.included ? <CheckCirledIcon /> : <CloseCirledIcon />}{' '}
-            <span className='ml-2'>{feature.description}</span>
+            <div className='relative left-5 top-1 h-4 w-4 rounded-full bg-white'>.</div>
```
Comment (egiegarciano14) [inline src/pages/components/subscriptionSection/PlanCard.js]:
```suggestion
            <div className='relative left-5 top-1 h-4 w-4 rounded-full bg-white'/>
```

Pls remove the tiny dot, so I think you can do it like this or if you have any other idea :bow:

Comment (egiegarciano14) [inline src/pages/components/subscriptionSection/PlanCard.js:14]:
```suggestion
            <div className='relative left-5 top-1 h-4 w-4 rounded-full bg-white'>.</div>
```
```suggestion
            <div className='relative left-5 top-1 h-4 w-4 rounded-full bg-white'/>
```

Pls remove the tiny dot, so I think you can do it like this or if you have any other idea :bow:

File: src/pages/components/tipSection/Tips.js [hunk 2]
```diff
@@ -7,12 +7,15 @@ const Tips = () => (
     <ul className='mb-7 list-none px-9 py-2'>
       {tips.map((tip, index) => (
         <li className='flex flex-row py-4' key={index}>
+          <div className='relative left-5 top-1 h-4 w-4 rounded-full bg-white'>.</div>
```
Comment (egiegarciano14) [inline src/pages/components/tipSection/Tips.js]:
```suggestion
          <div className='relative left-5 top-1 h-4 w-4 rounded-full bg-white'/>
```

**Review by egiegarciano14 — APPROVED** [review]

**Pull request merged**